installer/conf/container.conf

```diff
@@ -41,6 +49,27 @@
   log_level info
 </filter>
 
+<filter oms.mdm.container.perf.telegraf**>
+  type filter_telegraf2mdm
+  custom_metrics_azure_regions eastus,southcentralus,westcentralus,westus2,southeastasia,northeurope,westeurope,southafricanorth,centralus,northcentralus,eastus2,koreacentral,eastasia,centralindia,uksouth,canadacentral,francecentral,japaneast,australiaeast
+  log_level debug
+</filter>
+
+<match oms.mdm.container.perf.telegraf**>
```

Comment:
Add this tag to existing output plugin and can add it back when more metrics are added for thread optimizaiton